etl/src/energuide/embedded/heating.py

```diff
@@ -120,15 +120,15 @@ def _get_equipment_type(cls, node: element.Element) -> bilingual.Bilingual:
 
     @staticmethod
     def _get_steady_state(node: element.Element) -> str:
-        steady_state_value = node.xpath('Type1/*/Specifications/@isSteadyState')[0]
+        steady_state_value = node.get('Type1/*/Specifications/@isSteadyState', str)
         return 'Steady State' if steady_state_value == 'true' else 'AFUE'
 
     @classmethod
     def from_data(cls, node: element.Element) -> 'Heating':
         return Heating(
             label=node.get_text('Label'),
             output_size=float(Heating._get_output_size(node)),
-            efficiency=float(node.xpath('Type1/*/Specifications/@efficiency')[0]),
+            efficiency=node.get('Type1/*/Specifications/@efficiency', float),
             steady_state=cls._get_steady_state(node),
             heating_type=cls._get_heating_type(node),
             energy_source=cls._get_energy_source(node),
```

Comment:
`heating.py` didn't have a try / except block that passed the error along as an `InvalidEmbeddedDataTypeError`. Should we add that?

Reply:
I am aware, it was/is on my to-do list